src/main/kotlin/org/rust/lang/core/psi/RsCodeFragmentFactory.kt

```diff
@@ -20,10 +22,17 @@ class RsCodeFragmentFactory(val project: Project) {
     private val psiFactory = RsPsiFactory(project, markGenerated = false)
 
     fun createCrateRelativePath(pathText: String, target: CargoWorkspace.Target): RsPath? {
-        check(pathText.startsWith("::"))
+        require(!pathText.startsWith("::")) { "Path shouldn't start with `::`" }
+        require(!pathText.startsWith("crate::")) { "Path shouldn't start with `crate::`" }
+
+        val prefix = when (target.edition) {
```

Comment:
Looks like this isn't needed

Reply:
Do you think that path without `::` prefix is enough?

Reply:
I do it this way in #3828 and looks like it works